corehq/apps/reports/standard/inspect.py

```diff
@@ -65,7 +65,11 @@ def _get_users_filter(self, mobile_user_and_group_slugs):
                                        mobile_user_and_group_slugs,
                                        self.request.couch_user)
                     .values_list('_id', flat=True))
-        return form_es.user_id(user_ids)
+        # If no filters are selected, return all results
```

Comment:
This returns all results if no users match the selected filters, which is different removing all options from the filter.  For instance, if the user selected a group which has no users, the report would show all forms, whereas I'd expect it to show no forms.

Rather, I think this should first check if any `mobile_user_and_group_slugs` were selected, and if not, skip the user_ids filter entirely, or return `match_all()`